responses/__init__.py

```diff
@@ -26,16 +26,18 @@
 from warnings import warn
 
 import yaml
-from requests.adapters import HTTPAdapter
-from requests.adapters import MaxRetryError
-from requests.exceptions import ConnectionError
-from requests.exceptions import RetryError
 
 from responses.matchers import json_params_matcher as _json_params_matcher
 from responses.matchers import query_string_matcher as _query_string_matcher
 from responses.matchers import urlencoded_params_matcher as _urlencoded_params_matcher
 from responses.registries import FirstMatchRegistry
 
+from ._compat import MOCKED_LIBRARY
```

Comment:
After thinking about it, I did not find a perfect way to support Niquests "perfectly" without doing the compat import switch.
I did found that this way offer more stability.